src/backend/vulkan/src/device.rs

```diff
@@ -1621,34 +1652,42 @@ impl d::Device<B> for Device {
                 .collect::<Vec<_>>(),
```

Comment:
still `Vec` here since `DescriptorSetLayout` wants `Arc<Vec>` :thinking:

Reply:
that's fine, creating descriptor set layouts is basically init time